### ISSUE TITLE
Could you provide the training configuration for pi05?

### ISSUE THREAD

**Issue by MZYzhouyi** [jingyu198/HUGE-Bench#8]
# Issue Description
I attempted to train on HUGE‑Bench by directly adopting the original pi05 training configuration from OpenPI. However, the resulting training performance is far worse even compared with pi0. I used the following configuration in `openpi/src/openpi/training/config.py`:
```python
TrainConfig(
    name="pi05_overall",
    model=pi0_config.Pi0Config(
        pi05=True,
        action_horizon=20,     # Adjust the action horizon to match your dataset.
        discrete_state_input=False,
    ),
    data=MyDroneDataConfig(
        repo_id="/dataset/HUGE_Dataset_v0/train",
        assets=AssetsConfig(asset_id="huge_overall"),
        base_config=DataConfig(
            prompt_from_task=True,
        ),
        extra_delta_transform=False,
    ),
    weight_loader=weight_loaders.CheckpointWeightLoader(
       "/weight/pi05_base"
    ),
    pytorch_weight_path="/weight/pi05_base",
    num_train_steps=25000,
    batch_size=512, #64 per gpu
    save_interval=200,
    fsdp_devices=8,
),
```
No other parts were modified. Could you please provide the correct training configuration?

**Comment by jingyu198** [comment]
Thanks for the detailed report. The original OpenPI Pi0.5 configuration is not directly sufficient for HUGE-Bench, because our experiments use the HUGE-Bench-specific data pipeline and policy mapping.

We have added the reference Pi0.5 training configuration and reproduction guide here:

https://github.com/jingyu198/HUGE-Bench/blob/main/docs/pi05_training.md

Please use the HUGE-Bench versions of `drone_policy.py` and `MyDroneDataConfig`, the `task_overall/train` split, and the `pi05_overall` configuration. The reference setup uses:

- `pi05=True`
- `action_horizon=20`
- `discrete_state_input=False`
- 25,000 training steps
- global batch size 512
- 8 FSDP devices

Before training, please run:

uv run scripts/compute_norm_stats.py --config-name pi05_overall

When loading an OpenPI Orbax `params` checkpoint, use `CheckpointWeightLoader` and leave `pytorch_weight_path` unset.

The fine-tuned Pi0.5 checkpoint is available here:

https://huggingface.co/yu781986168/HUGE_PI05

Results may vary depending on the OpenPI revision, dataset version, normalization assets, GPU configuration, and random seed.
